caproto/asyncio/server.py

```diff
@@ -26,9 +26,13 @@ async def wait(self, timeout=None):
         return self.is_set()
 
 
+# Back-compat
+Event = AsyncioEvent
+
+
 class AsyncioAsyncLayer(AsyncLibraryLayer):
     name = 'asyncio'
-    Event = asyncio.Event
```

Comment:
This looks like we did not use the custom Event here though?

Reply:
Right, that was a mistake at the time of the asyncio server PR.

The sole difference here is just that the `timeout` kwarg will now actually async-lib agnostic (between curio, trio, and asyncio).